test/item_repository_test.rb

```diff
@@ -48,50 +46,61 @@ def test_random
 
   def test_find_by_item_id
     item = item_repository.find_by_item_id(7)
+
     assert_equal 7, item.id
   end
 
   def test_find_by_name
-    item = item_repository.find_by_name('Item')
-    assert_equal 'Item', item.name
+    item = item_repository.find_by_name("Item")
+
+    assert_equal "Item", item.name
   end
 
   def test_find_by_item_description
-    item = item_repository.find_by_item_description('Autem Minima,Cumque consequuntur ad.')
-    assert_equal 'Autem Minima,Cumque consequuntur ad.', item.description
+    item = item_repository
+      .find_by_item_description("Autem Minima,Cumque consequuntur ad.")
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.